src/librustc/diagnostics.rs

```diff
@@ -329,6 +329,49 @@ loop. Without a loop to break out of or continue in, no sensible action can be
 taken.
 "##,
 
+E0271: r##"
```

Comment:
Erf, it's because I wasn't on the good branch and copy/paste my code. I just forgot to remove the entry in the `register_diagnostics`. I fix it.